fs/sync/sync.go

```diff
@@ -61,6 +61,8 @@ type syncCopyMove struct {
 	trackRenamesWg sync.WaitGroup         // wg for background track renames
 	trackRenamesCh chan fs.Object         // objects are pumped in here
 	renameCheck    []fs.Object            // accumulate files to check for rename here
+	CompareDest    fs.Fs                  // place to check for files
```

Comment:
These should be small initial letter so they aren't exported, so `compareDest`, `copyDest`